src/wallet/rpcwallet.cpp

```diff
@@ -2497,6 +2503,216 @@ UniValue fundrawtransaction(const UniValue& params, bool fHelp)
     return result;
 }
 
+UniValue getlabeladdress(const UniValue& params, bool fHelp)
```

Comment:
would `getnewlabeladdress` be wrong? As-is gave me the impression that this was some static address.